lib/timeutils/misc.c

```diff
@@ -142,3 +144,23 @@ timespec_diff_nsec(struct timespec *t1, struct timespec *t2)
 {
   return (glong)((t1->tv_sec - t2->tv_sec) * 1e9) + (t1->tv_nsec - t2->tv_nsec);
 }
+
+void timeutils_setup_timezone_hook(void);
+
+static void timeutils_reset_timezone(gint type, gpointer context)
+{
+  (void) type;
+  (void) context;
+
```

Comment:
please either add G_GNUC_UNUSED to the arguments, or remove these lines. I don't think the compile would warn about these.